test/controllers/patients_controller_test.rb

```diff
@@ -12,6 +15,65 @@ class PatientsControllerTest < ActionDispatch::IntegrationTest
     @clinic = create :clinic
   end
 
+  describe 'index method' do
+    before do
+      # Not sure if there is a better way to do this, but just calling
+      # `sign_in` a second time doesn't work as expected
+      delete destroy_user_session_path
+    end
+
+    it 'should reject users without access' do
+      sign_in @user
+
+      get patients_path
+      assert_redirected_to root_path
+
+      get patients_path + '.csv'
```

Comment:
I haven't had to do this yet, but I THINK the new way to do this is by strapping `as: :csv` at the end, instead of the string concat. So something like `get patients_path, as: :csv`. See http://api.rubyonrails.org/classes/ActionDispatch/IntegrationTest.html (control-f 'Changing the request encoding).